csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests/runtest-docker-gpu.sh

```diff
@@ -38,7 +38,6 @@ docker run -h $HOSTNAME \
         --name "onnxruntime-gpu-container" \
         --volume "$SOURCE_ROOT:/onnxruntime_src" \
         --volume "$BUILD_DIR:/home/onnxruntimedev" \
-        --volume "$HOME/.cache/onnxruntime:/home/onnxruntimedev/.cache/onnxruntime" \
```

Comment:
Will removing this result in it downloading the 25GB test models file each time? IIRC it will check this cache directory first for the zip file and only download if it's missing from there.

Reply:
The downloading should only take 2 minutes, and we are running out of disk space, so we can't cache them.